shared/src/packages/chats/libs/types/chat-get-all-item-response-dto.type.ts

```diff
@@ -0,0 +1,9 @@
+import { type UserToChatResponseDto } from './user-to-chat-response-dto.type.js';
+
+type ChatGetAllItemResponseDto = {
+  id: number;
+  name: string;
+  members: UserToChatResponseDto[];
```

Comment:
define properties directly here and remove UserToChatResponseDto
```suggestion
  members: {...}[];
```